src/components/nodes/subProcess/index.js

```diff
@@ -39,8 +39,8 @@ export default {
 
     setNodeProp(node, 'calledElement', currentConfig.calledElement);
 
-    if (currentConfig.name != value.name) {
-      if (oldConfig.name === value.name || !oldConfig.name) {
+    if (currentConfig.name !== value.name) {
+      if (currentConfig.name && (oldConfig.name === value.name || !oldConfig.name)) {
```

Comment:
I find this block a little hard to read. I wonder if it can be pulled out into a function and then refactored.

Reply:
I wasn't 100% sure what it is doing to give it a name.

Reply:
Random name suggestions, feel free to ignore:

`currentConfig.name !== value.name` ==> `nameChanged`

`currentConfig.name && (oldConfig.name === value.name || !oldConfig.name)` ==> `nameChangedByFormSelect`

Or something like that, not sure my suggestion is an improvement :)